README.md

```diff
@@ -2,3 +2,13 @@
 
 Drupal 8 Packaging/Compile tasks for Robo Task Runner
 
+[![Latest Stable Version](https://poser.pugx.org/digipolisgent/robo-digipolis-package-drupal8/v/stable)](https://packagist.org/packages/digipolisgent/robo-digipolis-package-drupal8)
+[![Latest Unstable Version](https://poser.pugx.org/digipolisgent/robo-digipolis-package-drupal8/v/unstable)](https://packagist.org/packages/digipolisgent/robo-digipolis-package-drupal8)
+[![Total Downloads](https://poser.pugx.org/digipolisgent/robo-digipolis-package-drupal8/downloads)](https://packagist.org/packages/digipolisgent/robo-digipolis-package-drupal8)
+[![PHP 7 ready](http://php7ready.timesplinter.ch/digipolisgent/robo-digipolis-package-drupal8/develop/badge.svg)](https://travis-ci.org/digipolisgent/robo-digipolis-package-drupal8)
+[![License](https://poser.pugx.org/digipolisgent/robo-digipolis-package-drupal8/license)](https://packagist.org/packages/digipolisgent/robo-digipolis-package-drupal8)
+
+[![Build Status](https://travis-ci.org/digipolisgent/robo-digipolis-package-drupal8.svg?branch=develop)](https://travis-ci.org/digipolisgent/robo-digipolis-package-drupal8)
+[![SensioLabsInsight](https://insight.sensiolabs.com/projects/7520a070-4500-494e-80c8-e0b109fb3db6/mini.png)](https://insight.sensiolabs.com/projects/7520a070-4500-494e-80c8-e0b109fb3db6)
+[![Code Climate](https://codeclimate.com/github/digipolisgent/robo-digipolis-package-drupal8/badges/gpa.svg)](https://codeclimate.com/github/digipolisgent/robo-digipolis-package-drupal8)
+[![Test Coverage](https://codeclimate.com/github/digipolisgent/robo-digipolis-package-drupal8/badges/coverage.svg)](https://codeclimate.com/github/digipolisgent/robo-digipolis-package-drupal8/coverage)
```

Comment:
Sowieso moeten we in de READMe.md nog uitleggen welke structuur verwacht wordt en verwijzen naar drupal-project als basis best!